collator/stop-collator.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+set +x
+
+docker-compose down
```

Comment:
I think we don't need a script for a single command?

Reply:
Removed